controller/cache/compliance.go

```diff
@@ -275,6 +310,11 @@ func (m CacheMethod) GetRiskScoreMetrics(acc, accCaller *access.AccessControl) *
 
 		if wl.ShareNetNS == "" {
 			epMap[cache.workload.ID] = &wlMini{mode: mode}
+			if _, ok := podVulsMap[cache.podName]; ok {
```

Comment:
why not use 
`if podVuls, ok := podVulsMap[cache.podName]; ok`
so that podVuls is referenced in the following 3 lines?

Reply:
fixed